### PR TITLE
chore(sdk): reuse existing tab instead of creating a new one

### LINKED ISSUE #159: [Bug]: hmr hot update will reopen a new window.
### Version

```sh
@rsdoctor/webpack-plugin: 0.1.0
@rsdoctor/rspack-plugin: 0.1.0
```

### Details

When dev mode, hmr hot update will reopen a new window, but the port is the same.

### PR DESCRIPTION
**Pull request by SoonIter**
## Related Links
closes #159 

## Checklist

- [x] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/sdk/src/sdk/utils/openBrowser.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    }
+  }
+
+  // Fallback to open
+  // (It will always open new tab)
+  try {
+    await open(url);
+    return true;
+  } catch (err) {
+    logger.error('Failed to open start URL.');
```
Comment (chenjiahan) [inline packages/sdk/src/sdk/utils/openBrowser.ts:79]: more appropriate to use Rsdoctor here

```suggestion
    logger.error('Failed to open Rsdoctor URL.');
```

File: packages/sdk/src/sdk/utils/openBrowser.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          {
+            cwd: join(__dirname, '../../../../static'),
+          },
+        );
+
+        return true;
+      }
+      logger.debug('Failed to find the target browser.');
+    } catch (err) {
+      logger.debug('Failed to open start URL with apple script.');
```
Comment (chenjiahan) [inline packages/sdk/src/sdk/utils/openBrowser.ts:68]:
```suggestion
      logger.debug('Failed to open Rsdoctor URL with apple script.');
```

File: packages/sdk/src/sdk/utils/openBrowser.ts (hunk 1 above)
Comment (SoonIter) [reply packages/sdk/src/sdk/utils/openBrowser.ts]: +1, fixed now

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
👍🏻👍🏻

**Pull request merged**